test/test_error_handlers.py

```
@@ -0,0 +1,74 @@
+        channel = FakeChannel(
+            connections_factory=BrokenConnectionFactory(self.listeners_handler_mock, self.queues_handler_mock)
+        )
+        my_listener = channel.add_listener()
+        my_listener.start()
+
+        # act
+        msg = Message(msg_type='test')
+        failed: bool = False
+        try:
+            channel.dispatch(listener_id=my_listener.id, msg=msg)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)